pkg/apis/crd/v1alpha1/types.go

```diff
@@ -409,6 +409,8 @@ type Rule struct {
 	// destinations.
 	// +optional
 	ToServices []NamespacedName `json:"toServices,omitempty"`
+	// TODO: comment here
```

Comment:
Please update doc as well when you update the comment for this new field.

Reply:
Comment added, will update the doc PR accordingly